storage/indexer/src/db_v2.rs

```diff
@@ -0,0 +1,407 @@
+}
+
+impl IndexerAsyncV2 {
+    pub fn open(
+        db_root_path: impl AsRef<std::path::Path>,
+        rocksdb_config: RocksdbConfig,
+        pending_on: DashMap<TableHandle, DashSet<Bytes>>,
+    ) -> Result<Self> {
+        let db_path = db_root_path.as_ref().join(INDEX_ASYNC_V2_DB_NAME);
+
+        let db = DB::open(
+            db_path,
+            "index_asnync_v2_db",
+            column_families(),
+            &gen_rocksdb_options(&rocksdb_config, false),
+        )?;
+
+        let next_version = db
+            .get::<IndexerMetadataSchema>(&MetadataKey::LatestVersion)?
+            .map_or(0, |v| v.expect_version());
+
+        Ok(Self {
+            db,
+            next_version: AtomicU64::new(next_version),
+            pending_on,
+        })
+    }
+
+    pub fn index(
+        &self,
+        db_reader: Arc<dyn DbReader>,
+        first_version: Version,
+        write_sets: &[&WriteSet],
+    ) -> Result<()> {
+        let last_version = first_version + write_sets.len() as Version;
+        let state_view = DbStateView {
+            db: db_reader,
+            version: Some(last_version),
+        };
+        let resolver = state_view.as_move_resolver();
+        let annotator = MoveValueAnnotator::new(&resolver);
+        self.index_with_annotator(&annotator, first_version, write_sets)
+    }
+
+    pub fn handle_pending_on_items(
+        &self,
+        db_reader: Arc<dyn DbReader>,
+        last_version: Version,
+    ) -> Result<()> {
+        let state_view = DbStateView {
+            db: db_reader,
+            version: Some(last_version),
+        };
+        let resolver = state_view.as_move_resolver();
+        let annotator = MoveValueAnnotator::new(&resolver);
+        let mut table_info_parser = TableInfoParser::new(self, &annotator, &self.pending_on);
+        table_info_parser.parse_pending_on()
+    }
+
+    pub fn index_with_annotator<R: MoveResolver>(
+        &self,
+        annotator: &MoveValueAnnotator<R>,
+        first_version: Version,
+        write_sets: &[&WriteSet],
+    ) -> Result<()> {
+        let end_version = first_version + write_sets.len() as Version;
+        let mut table_info_parser = TableInfoParser::new(self, annotator, &self.pending_on);
+        for write_set in write_sets {
+            for (state_key, write_op) in write_set.iter() {
+                table_info_parser.parse_write_op(state_key, write_op)?;
+            }
+        }
+        let mut batch = SchemaBatch::new();
+        match table_info_parser.finish(&mut batch) {
+            Ok(_) => {},
+            Err(err) => {
+                aptos_logger::error!(first_version = first_version, end_version = end_version, error = ?&err);
+                write_sets
+                    .iter()
+                    .enumerate()
+                    .for_each(|(i, write_set)| {
+                        aptos_logger::error!(version = first_version as usize + i, write_set = ?write_set);
+                    });
```

Comment:
I think that we can actually remove this. It was used for debugging something as I remember these are too noisy.